src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -927,4 +927,12 @@ public List<LibraryUsage> findLibraryUsage(String libraryName) {
     }
     return measureRepository.findLibraryUsageByLibraryName(libraryName);
   }
+
+  public int countAllMeasures() {
+    return measureRepository.countAllByActive(true);
+  }
+
+  public int countMyMeasures(String user) {
+    return measureRepository.countAllMyMeasures(true, user);
```

Comment:
The story says 

Add a number (in parentheses) in the My Measures tab that displays the number of measure sets in the tab (that is the number of measures at the top level of the nested measure hierarchy that was implemented in [MAT-8032](https://jira.cms.gov/browse/MAT-8032))

So I have 2 questions, 
1. Shouldn't we only send back number of unique measure sets ?
2. this is more of front end, but shouldn't we be displaying the number of measures in the tab ( i.e do we consider the pagination ) ?

Reply:
1. Oh geez you're right. I took it as the total number of measures. Fixing now.
2. That one I'm not so sure. Based just on the example image I'd say no and its just the total unique sets

Reply:
I agree it seems like it, just wanted to check

Reply:
Changed this around so that were looking at the sets